src/Utils/Responsive/index.ts

```diff
@@ -0,0 +1,26 @@
+export * from "./SuperDeprecatedResponsive"
+import * as theme from "@artsy/palette"
+import { createMedia } from "./Media"
+
+// We need this to be 0-based, whereas currently in palette xs is defined as 767
+const newThemeBreakpoints = {
+  xs: 0,
+  sm: 768,
+  md: 900,
+  lg: 1024,
+  xl: 1192,
+}
```

Comment:
We should update Palette with these new values an publish a new release, i think it will be fine.

Reply:
I guess we can update the media query for the `xs` breakpoint so that it won’t break existing `Responsive` usage, but do you think we’re ok with styled-system using a different value for `xs`?

Reply:
Oh shoot, I read this wrong. 🤔 Yeah, might be safest to return to this in another PR. Maybe add a FIXME around uniting this with Palette? We def need to make sure that styled-system, palette and Reaction (and other consumers of each) are all on the same page.